WordPress/gutenberg
# Comments Query Loop: Update default template

## What?

Changes the default template to make it more aesthetic. Also, it modifies the default values of some block attributes.

## Why?

To let editor users have a first good structure for the Comments Query Loop inner blocks.

## How?

- Changing the initial `innerBlocks` for the Comments Query Loop block.
- Changing the default value of `isLink` and `fontSize` attributes of
   - Comment Author Name
   - Comment Date
   - Comment Edit Link
   - Comment Reply Link

## Testing Instructions

1. Open a post or page
2. Add a Comments Query Loop block
3. Check that the default block organization looks like in the image below.

## Screenshots or screencast 

![Screenshot 2022-04-07 at 17 05 35](https://user-images.githubusercontent.com/6917969/162245935-77a62744-7a02-4089-bcea-eb4356e14357.png)
